### ISSUE TITLE
freesurfer reconall outputs missing

### ISSUE THREAD

**Issue by JailanOweda** [nipy/nipype#3667]
In the freesurfer interface when using the reconall function, not all files generated in the cache_directory are listed as possible outputs that you can use as inputs for the next node (e.g. copy them to the derivatives directory using datasink).
E.g. none of the files in the scripts directory are listed.

**Comment by effigies** [comment]
Yes, the FreeSurfer interface assumes you're looking for imaging outputs. To add outputs, patch here:

https://github.com/nipy/nipype/blob/2e36f69a833529ee8b66c1c3fc43746db5a9fcea/nipype/interfaces/io.py#L1634-L1840
